src/Years/index.js

```diff
@@ -142,10 +142,17 @@ export default class Years extends Component {
       ? rowHeight + SPACING
       : rowHeight
     );
-    const containerHeight = years.length * rowHeight < height + 50
+    const isYearLess = years.length * rowHeight < height + 50;
```

Comment:
What is the meaning of `isYearLess`? Sounds like a boolean value, but it's a number.

Reply:
`isYearLess` is a boolean value  which means the  `all years rows height`, is less than `component  prop height`.